src/services/extensions/links.ts

```diff
@@ -276,3 +285,18 @@ export function escapeUrlPipeDelimiters(value: string | undefined): string {
 export function escapeURLDelimiters(value: string | undefined): string {
   return escapeUrlCommaDelimiters(escapeUrlPipeDelimiters(value));
 }
+
+// Copied from interpolateQueryExpr in loki datasource, as we can't return a promise in the link extension config we can't fetch the datasource from the datasource srv, so we're forced to duplicate this method
+export function interpolateQueryExpr(value: string | unknown[], variable: QueryVariableModel | CustomVariableModel) {
+  // if no multi or include all do not regexEscape
+  if (!variable.multi && !variable.includeAll) {
+    return value;
+  }
+
+  if (typeof value === 'string') {
+    return escapeLabelValueInExactSelector(value);
+  }
+
+  const escapedValues = lodashMap(value, lokiSpecialRegexEscape);
```

Comment:
Is map from lodash really needed for this?

Reply:
We get it for free from grafana afaik, it's not in the bundle when I checked, and otherwise I just copied the method from core

Reply:
Yeah, I mean a regular array map should do the same? Not really important anyway, but I was wondering about the reason to use it.

Reply:
yeah i think it's just a style thing